apps/ledger-live-desktop/src/newArch/features/Collectibles/Nfts/BreadCrumb/useBreadCrumbModel.tsx

```diff
@@ -7,16 +7,30 @@ import { useCallback, useMemo } from "react";
 import { ProtoNFT } from "@ledgerhq/types-live";
 import { DropDownItemType } from "~/renderer/components/DropDownSelector";
 import { setTrackingSource } from "~/renderer/analytics/TrackPage";
+import { useFeature } from "@ledgerhq/live-common/featureFlags/index";
+import { isThresholdValid, useNftGalleryFilter } from "@ledgerhq/live-nft-react";
 
 const useBreadCrumbModel = () => {
   const history = useHistory();
   const { id, collectionAddress } = useParams<{ id?: string; collectionAddress?: string }>();
+  const nftsFromSimplehashFeature = useFeature("nftsFromSimplehash");
+  const thresold = nftsFromSimplehashFeature?.params?.threshold;
 
   const account = useSelector((state: State) =>
     id ? accountSelector(state, { accountId: id }) : null,
   );
 
-  const collections = useMemo(() => nftsByCollections(account?.nfts), [account?.nfts]);
+  const { nfts } = useNftGalleryFilter({
+    nftsOwned: account?.nfts || [],
+    addresses: String(account?.freshAddress),
+    chains: [String(account?.currency.id)],
+    threshold: isThresholdValid(thresold) ? Number(thresold) : 75,
```

Comment:
Is there a reason not to manage the threshold with FF behavior directly into the hook, please? It seems that every time we use this hook we have to copy past this logic.

Reply:
I found the reason => It's in a library